tests/integration/components/move-file-modal/component-test.ts

```diff
@@ -172,4 +175,83 @@ module('Integration | Component | move-file-modal', hooks => {
+        const projectModel = server.create('node', 'withFiles', 'withStorage', 'currentUserAdmin');
+
+        const osfStorageModel = projectModel.files.models[0];
+        const folderModel = server.create('file', {
+            target: projectModel,
+            parentFolder: osfStorageModel.rootFolder,
+        }, 'asFolder');
+        const folder = await this.store.findRecord('file', folderModel.id);
+
+        const project = await this.store.findRecord('node', projectModel.id);
+        const providers = await project.queryHasMany('files');
+        const osfStorage = providers.findBy('id', osfStorageModel.id);
+
+        const queueFileModel = server.create('file', {
+            target: projectModel,
+            parentFolder: osfStorageModel.rootFolder,
+        });
+        const queueFile = new OsfStorageFile(this.currentUser, await this.store.findRecord('file', queueFileModel.id));
+        const storageCapFileModel = server.create('file', {
+            target: projectModel,
+            parentFolder: osfStorageModel.rootFolder,
+        });
+        const storageCapFile = new OsfStorageFile(
+            this.currentUser, await this.store.findRecord('file', storageCapFileModel.id),
+        );
+        const nameConflictFileModel = server.create('file', {
+            target: projectModel,
+            parentFolder: osfStorageModel.rootFolder,
+        });
+        const nameConflictFile = new OsfStorageFile(
+            this.currentUser, await this.store.findRecord('file', nameConflictFileModel.id),
+        );
+
+        server.namespace = '/wb';
+        server.post(`/files/${queueFile.id}/move`, () => ({
+            data: 'queued', status: 202,
+        }), 202);
+        server.post(`/files/${storageCapFileModel.id}/move`, () => ({ message: 'over storage'}), 507);
+        server.post(`/files/${nameConflictFileModel.id}/move`, () => ({ message: 'name conflict'}), 409);
+
+        const parentFolder = new OsfStorageProviderFile(this.currentUser, osfStorage!);
+        const currentFolder = new OsfStorageFile(this.currentUser, folder);
+        const manager = {
+            targetNode: await this.store.findRecord('node', project.id),
+            currentFolder,
+            folderLineage: [parentFolder, currentFolder],
+            goToFolder: sinon.fake(),
+        };
+        this.manager = manager;
+        this.close = sinon.fake();
+        this.filesToMove = [queueFile, storageCapFile, nameConflictFile];
+
+        await render(hbs`
+        <MoveFileModal
+            @isOpen={{true}}
+            @close={{this.close}}
+            @preserveOriginal={{false}}
+            @filesToMove={{this.filesToMove}}
+            @manager={{this.manager}}
+        />
+        `);
+
+        await click('[data-test-breadcrumb="osfstorage"]');
+        await click('[data-test-move-files-button]');
+
+        assert.dom(`[data-test-moving-file-item=${queueFile.id}]`).hasText(
```

Comment:
Should this be more like

```suggestion
        assert.dom(`[data-test-moving-file-item="${queueFile.id}"]`).hasText(
```

I don't know why that would fail probabilistically, but it's how we tend to do them in tests.

Reply:
That is a weird quirk... I don't understand why it doesn't care about the quotes in some, but not other scenarios. I had assumed something fishy was going on with how 400-type errors were handled in tests, so I'm glad that's not the rabbit-hole I need to be going down.